apps/src/gamelab/GameLabSprite.js

```diff
@@ -235,7 +235,7 @@ var AABBops = function (p5Inst, type, target, callback) {
         print("busted");
         return false;
       }*/
-      if(this.collider != undefined && other.collider != undefined)
+      if(this.collider !== undefined && other.collider !== undefined)
```

Comment:
I think this might be a subtle behavior change?  This would have matched null before, and now it won't.